faker/providers/address/__init__.py

```diff
@@ -93,8 +93,10 @@ def geo_coordinate(cls, center=None, radius=0.001):
         if not center:
             return Decimal(str(random.randint(-180000000, 180000000) / 1000000.0)).quantize(Decimal('.000001'))
         else:
+            center = float(center)
+            radius = float(radius)
             geo = random.uniform(center - radius, center + radius)
```

Comment:
Sure thing, it's PR #281 

This is probably a bad place to ask (I'm new to Github), but does this method need some tests added since this seems to decrease the coverage/coveralls?